src/services/analytics/constants.ts

```diff
@@ -0,0 +1,42 @@
+import Config from "react-native-config";
+
+// -----------------------------------------------------------------------------
+// API CONFIGURATION
+// -----------------------------------------------------------------------------
+
+export const AMPLITUDE_API_KEY = Config.AMPLITUDE_API_KEY ?? "";
+
+// -----------------------------------------------------------------------------
+// STORAGE KEYS
+// -----------------------------------------------------------------------------
+
+export const STORAGE_KEYS = {
+  METRICS_USER_ID: "metrics_user_id",
+} as const;
+
+// -----------------------------------------------------------------------------
+// TIMING CONSTANTS
+// -----------------------------------------------------------------------------
+
+export const TIMING = {
+  THROTTLE_DELAY_MS: 500,
+} as const;
+
+// -----------------------------------------------------------------------------
+// DEBUG CONFIGURATION
+// -----------------------------------------------------------------------------
+
+export const DEBUG_CONFIG = {
+  MAX_RECENT_EVENTS: 50,
+  LOG_PREFIX: "Analytics",
+} as const;
+
+// -----------------------------------------------------------------------------
+// ANALYTICS CONFIGURATION
+// -----------------------------------------------------------------------------
+
+export const ANALYTICS_CONFIG = {
+  DEFAULT_ENABLED: false,
+  INCLUDE_PLATFORM_CONTEXT: true,
```

Comment:
```suggestion
  INCLUDE_COMMON_CONTEXT: true,
```

I'd suggest naming it like this as it includes more than just platform info